File: Indixpert-Sept-batch2025/C Languege/function/task4.c
#include<stdio.h>
int add(int,int);
void main()
{
    int a , b , c;

    printf("Enter Your a Number = ");
    scanf("%d",&a);

    printf("Enter Your b Number = ");
    scanf("%d",&b);

    c = add(a,b);

    printf("%d",c);

}
int add(int a, int b)
{
    int c ;
    c = a + b;

    return c;

}

File: Indixpert-Sept-batch2025/Python/List/task4.py
list=[]
num=int(input("Enter your number ="))
for i in range(num):
    n=int(input("Enter Element ="))
    list.append(n)

even_total=0
odd_total=0

for i in list:
    if i % 2 == 0:
        even_total = even_total+i
    elif i % 2 != 0:
        odd_total =odd_total+i

print("sum of Even number =",even_total)
print("sum of odd number =",odd_total)